### PR TITLE
add new conferences

### PR DESCRIPTION
**Pull request by marandaneto**

### CODE REVIEW

File: _conferences/flutter-bytes-conf-november-2024.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+website: https://www.flutterbytesconf.com/
+location: IS COMING SOON...
+online: false
+
+date_start: 2023-11-01
+date_end:   2023-11-02
+
+cfp:
+  start: 2023-11-01
+  end:   2023-11-01
```
Comment (marandaneto) [inline _conferences/flutter-bytes-conf-november-2024.md:12]: TBD

File: _conferences/flutterconf-latam-october-2024.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+website: https://flutterconflatam.dev/
+location: Arequipa, Perú
+online: false
+
+date_start: 2024-10-29
+date_end:   2024-10-30
+
+cfp:
+  start: 2024-10-29
+  end:   2024-10-29
```
Comment (marandaneto) [inline _conferences/flutterconf-latam-october-2024.md:12]: TBD

File: _conferences/droidcon-kenya-november-2024.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+name: "droidcon Kenya"
+website: https://droidcon.co.ke/
+location: Kenya
```
Comment (marandaneto) [inline _conferences/droidcon-kenya-november-2024.md]:
```suggestion
location: Nairobi, Kenya
```

File: _conferences/flutter-bytes-conf-november-2024.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
+name: "Flutter Bytes Conference"
+website: https://www.flutterbytesconf.com/
+location: IS COMING SOON...
```
Comment (marandaneto) [inline _conferences/flutter-bytes-conf-november-2024.md]:
```suggestion
location: Lagos, Nigeria
```

**Comment by marandaneto** [comment]
https://sessionize.com/droidcon-london-2024 does not have any flutter interest apparently so not adding it

File: _conferences/droidcon-italy-november-2024.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+---
+name: "droidcon Italy"
+website: https://it.droidcon.com/2024/
+location: Milan - Italy
```
Comment (ueman) [inline _conferences/droidcon-italy-november-2024.md:4]: I always wanted to visit Italy, maybe this is the time? :D

**Review by ueman — APPROVED** [review]
LGTM

File: _conferences/droidcon-italy-november-2024.md (hunk 5 above)
Comment (marandaneto) [reply _conferences/droidcon-italy-november-2024.md:4]: I might go as well but waiting for the ticket prices =D
Italy is a must ^^

Comment (marandaneto) [reply _conferences/droidcon-italy-november-2024.md:4]: I am going btw, just bought the ticket.

Comment (ueman) [reply _conferences/droidcon-italy-november-2024.md:4]: I applied with 3 talks. Fingers crossed that one will get chosen 🤞

**Pull request merged**